src/workspace-data/WorkspaceAttributes.js

```diff
@@ -321,17 +396,6 @@ export const WorkspaceAttributes = ({
                                   },
                                   [icon('edit', { size: 19 })]
                                 ),
-                                h(
-                                  Link,
-                                  {
-                                    tooltip: 'Delete variable',
-                                    ...WorkspaceUtils.getWorkspaceEditControlProps(workspace),
-                                    style: { marginLeft: '1rem' },
-                                    onClick: () => setDeleteIndex(rowIndex),
-                                    'aria-haspopup': 'dialog',
-                                  },
-                                  [icon('trash', { size: 19 })]
-                                ),
```

Comment:
I don't mind leaving the delete icon in place on each row, especially since the edit icon is there already. Just an opinion; I'd defer to UX or Product if it makes more sense to omit it.

Reply:
The idea was to get closer to the data table UI, which has an edit button for the individual cell value vs the entire row.